$NetBSD: patch-aa,v 1.1.1.1 2008/04/06 23:03:23 seb Exp $

Sync with current Rake and drop the ruby-debug gem build.

--- Rakefile.orig	1970-01-01 00:00:00.000000000 +0000
+++ Rakefile
@@ -23,7 +23,6 @@ CLI_FILES = COMMON_FILES + FileList[
   "cli/**/*",
   'ChangeLog',
   'bin/*',
-  'doc/rdebug.1',
   'test/**/*.cmd',
   'test/**/*.right',
   'test/**/gcd.rb',
@@ -150,16 +149,13 @@ end
 Rake::GemPackageTask.new(base_spec) do |pkg|
   pkg.need_tar = true
 end
-Rake::GemPackageTask.new(cli_spec) do |pkg|
-  pkg.need_tar = true
-end
 
 task :default => [:package]
 
 # Windows specification
 win_spec = base_spec.clone
 win_spec.extensions = []
-win_spec.platform = Gem::Platform::WIN32
+win_spec.platform = Gem::Platform::CURRENT
 win_spec.files += ["lib/#{SO_NAME}"]
 
 desc "Create Windows Gem"
